### ISSUE TITLE
Advertised URL not working

### ISSUE THREAD

**Issue by cco3** [the-hardy-boys/hymnal-api#2]
http://hymnal-api.kingnebby.com/hymn

**Comment by kingnebby** [comment]
Ya, unfortunately I stopped the server. You wanted to build something around this?

**Comment by cco3** [comment]
Yeah, just want to display lyrics without hosting them.  I may use your buildDB script.
